UK/Data/Plugin/CCDS-R/Settings/conversions.json

```diff
@@ -0,0 +1,21 @@
+{
+    "labels": {
+		"7400": "UAVLINK",
+        "7500": "HIJACK",
+        "7600": "RTFAIL",
+        "7700": "SOS",
+        "0030": "LOST"
+    },
+    "callsigns": {
+		"0003": "HLE60",
+		"0014": "HLE21",
+		"0015": "HLE07",
+		"0016": "HLE24",
+		"0017": "HLE27",
+		"0031": "HLE55",
+		"0027": "FIRJOIN",
+        "1177": "FIS",
```

Comment:
As we'll be covering Scottish, I'd suggest putting the Scottish FIS code here as well (7401)